content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -144,36 +172,40 @@ Client: I am doing this for the first time
 Client: BYE
 Closing connection
 ```
-### Client-side Programming
+### Programming on the Client-side
 #### Establish a Socket Connection
-A Socket instance can be obtained in two ways: 
-    
-1. Server side receives it as a return value of the accept method.
-2. You can instantiate it as a Socket like so:
+There are two ways to get a Socket instance:
+
+1.Server receives the accept method as a return value.
```

Comment:
> 1.Server receives the accept method as a return value.

Add a space after the **1.**. I would also rephrase it to be more grammatically correct like so:

> 1. The server receives it as a return value of the accept method